### PR TITLE
Hotfix for a few Gemini/GMOS docs

### PR DESCRIPTION
**Pull request by profxj**
As titled

### CODE REVIEW

**Comment by profxj** [comment]
@Tang-SL -- If you have additional suggestions, please 
let me know or add them directly on this PR

File: doc/gemini_gmos.rst [hunk 1]
```diff
@@ -18,6 +18,23 @@ taken in Nod+Shuffle mode by simply replicating the calibrations.
 That is, we do *not* subtract the nodded images but reduce
 it as if it were a separate slit.
 
+For this mode, you need to specify the `gemini_gmos_north_ham_ns` 
+spectrograph.
+
+Slits
```
Comment (jhennawi) [inline doc/gemini_gmos.rst:24]: Are these the defaults in the spectrograph file for this instrument? If you do that, then it is not necessary to add to the pypeit file.

**Review by jhennawi — CHANGES_REQUESTED** [review]
Minor change request.

File: doc/gemini_gmos.rst (hunk 1 above)
Comment (profxj) [reply doc/gemini_gmos.rst:24]: These are not defaults and should not be as GMOS can have a slit mask.

We could try to make them defaults for the longslit if we can reliably
tell when it *is* the longslit.

**Review by Tang-SL — APPROVED** [review]

File: doc/gemini_gmos.rst (hunk 1 above)
Comment (jhennawi) [reply doc/gemini_gmos.rst:24]: Okay that is good. Please update the dev suite examples accordingly if there are any and add a comment explaining these lines.

**Review by jhennawi — APPROVED** [review]

**Pull request merged**